Exercises/arrays/arrays.js

```diff
@@ -0,0 +1,41 @@
+//1. Make a function which will generate array with random length and random values.
+function generateArray() {
+  const arrayRandom = Math.floor(Math.random() * 100);
+  const arrayValues = new Array(arrayRandom);
+  for (let i = 0; i < arrayValues.length; i++) {
+    arrayValues[i] = Math.floor(Math.random() * 10);
+  }
+  return arrayValues;
+}
+//console.log(generateArray());
+//2. Make a function which will take array and return copy of it.
+function arrayCopy(array) {
+  const results = [...array];
+  return results;
+}
+//console.log(arrayCopy([1,2]));
+//3. Make a function which will take array and element and return a index of element in array.
+function array(arr, element) {
+  const results = arr.indexOf(element);
+  return results === -1 ? 'Element is not found' : results;
+}
+//console.log(array([1,2,3], 7));
+//4. Make function which will take two arrays and return association of them.
+function associationArray(array1, array2) {
+  const results = array1.concat(array2);
+  return results;
```

Comment:
just `return arrayOne.concat(arrayTwo)` also not good to use numbers in variables